src/linux/gpu.rs

```diff
@@ -0,0 +1,220 @@
+#![allow(clippy::too_many_arguments)]
+
+use std::fs::{read_dir, File};
+use std::io::Read;
+use std::path::Path;
+
+use crate::GpuExt;
+
+#[doc = include_str!("../../md_doc/gpu.md")]
+pub struct Gpu {
+    name: String,
+    gpu_usage: Option<f32>,
+    vram_used: Option<u64>,
+    vram_total: Option<u64>,
+    freq: Option<u64>,
+    freq_max: Option<u64>,
+    vendor_id: String,
+    brand: Option<String>,
+}
+
+impl Gpu {
+    /// Creates a new instance of `Gpu` with everything set to `None`.
+    pub fn new() -> Self {
```

Comment:
What's the point of this method?

Reply:
It was useless, removed in 2f02dc85deda5a09445e1dc47cf58332662ed2fc